README.md

```diff
@@ -0,0 +1,32 @@
+Project 1 of Microverse 2.0 program: Positioning and Floating Elements
+
+We were asked to make a clone of a New York Times article page using the following concepts: Embedding images The box model Flexbox Positioning
+
+Project specifications can be found at https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements
+
+Project requirements: clean, code, semantic HTML, aside tag for editor's picks, use float, flex and grid techniques.
+
+
+![READMESNEAKPEAK](https://user-images.githubusercontent.com/56790126/108226304-75532f00-7145-11eb-9f71-7916903fbc67.png)
+
+Additional description about the project and its features.
+
+Built With
+HTML5&CSS3
+
+[Live Demo Link](https://amiraabouhadid.github.io/project1/)
+
+
+Authors
+
+Github: @amiraabouhadid
+Twitter: @AbouhadidAmira
+LinkedIn: @amira-abouhadid
+
+Contributing reviews, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a star if you like this project!
+
+Acknowledgments to anyone whose code was used Inspiration etc. License This project is MIT licensed.
```

Comment:
## Improve `README`
- [ ] This README can be improved further to appear more stylish in a professional way. Fell free to have look at this [template ](https://github.com/microverseinc/readme-template)and draw inspiration from there. 🧑‍💼